### DISCUSSION TITLE
ui_openMVG_control_points_registration

### DISCUSSION THREAD

**Discussion by maazraan** [openMVG/openMVG#2235] [Q&A] [answered]
Can someone please tell me the algorithm behind ui_openMVG_control_points_registration, cause instead of marking the points in the images using the ui i want to add the gcps via a formatted text file which includes a point's actual position and position on different images.

**Comment by pmoulon** [comment, marked as answer]
Details are shared here https://github.com/openMVG/openMVG/blob/develop/docs/sphinx/rst/software/ui/SfM/control_points_registration/GCP.rst
